playbooks/roles/lets-encrypt/tasks/main.yml

```diff
@@ -6,6 +6,16 @@
     state: directory
     recurse: yes
 
+- name: Query Let's Encrypt to find out the latest agreements
```

Comment:
Minor nit: I think this would be clearer as:
"Query Let's Encrypt to find the current subscriber agreement URL"